ux.symfony.com/src/Controller/MainController.php

```diff
@@ -29,4 +30,13 @@ public function homepage(UxPackageRepository $packageRepository): Response
             'recipeFileTree' => new RecipeFileTree(),
         ]);
     }
+
+    #[Route(path: '/robots.txt', name: 'app_robots')]
+    public function __invoke(Request $request): Response
+    {
+        $response = $this->render('robots.txt.twig');
+        $response->headers->set('Content-Type', 'text/plain');
+
+        return $response;
+    }
```

Comment:
What about creating the file in `public/`? 
So the file can be delivered by the webserver instead of the Symfony app

Reply:
Ah, because you use `{{ url('app_sitemap') }}`

Reply:
To be fair, you're right, and i think we will soon render it once after the deployment ... but i need to check first how it goes :)